README.md

```diff
@@ -1 +1,135 @@
 # crypto
+
+## First Time Setup
+
+1. Install PyEnv to manage python installations. `brew install pyenv`
+
+2. Add the following lines to your `~/.bash_profile`
+
+    ```bash
+
+    export PATH="~/.pyenv/bin:$PATH"
+
+    eval "$(pyenv init --path)"
+
+    eval "$(pyenv virtualenv-init -)"
+
+    ```
+
+3. Open a new terminal window or `source ~/.bash_profile`
+
+4. Install the latest version of Python using `pyenv install 3.9.6`
+
+5. Check if your system python is the same `python --version`
+
+6. Install poetry
+
+ 
+
+    ```bash
+
+    curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
```

Comment:
According to the docs get-poetry.py will soon be deprecated and if we want to install newer versions of poetry we should use install-poetry.py instead as described here https://python-poetry.org/docs/master/#installation

Reply:
Fixed!